Tests/test_simulation_packages_import.py

```diff
@@ -54,20 +54,23 @@ def test_bioscrape_import_simulate_via_sbml():
 
 def test_libroadrunner_import():
     from biocrnpyler import ChemicalReactionNetwork
+    CRN = ChemicalReactionNetwork(species=[], reactions=[])
     try:
-        import numpy as np
-
-        CRN = ChemicalReactionNetwork(species=[],reactions=[])
+        import roadrunner
+    except ModuleNotFoundError:
         with pytest.warns(None) as record:
-            sim_results = CRN.runsim_roadrunner(timepoints=np.linspace(0, 10, 100), filename=None)
+            sim_results = CRN.simulate_with_roadrunner(timepoints=list(range(0,10)))
 
         assert sim_results is None
 
         # only one warning was triggered
         assert len(record) == 1
         # check the warning message
         assert str(record[0].message) == "libroadrunner was not found, please install libroadrunner"
```

Comment:
I guess this warning print out is fine but I am wondering why the 6 line set up to do this and not just print out or throw a normal warning if ModuleNotFoundError?

Reply:
Good question! it's an older test which I'm using here, I'd write it differently today, it's a bit baroque indeed :)

Reply:
I went back and had a good look at it and simplified the code a bit, added some comments as well